prboom2/src/MUSIC/portmidiplayer.c

```diff
@@ -385,7 +388,7 @@ static void writesysex (unsigned long when, int etype, unsigned char *data, int
 static void pm_stop (void)
 {
   int i;
-  unsigned long when = Pt_Time ();
+  unsigned long when = 0;
```

Comment:
I guess it's safe to revert this change back to the original code.

Reply:
I think so too.